core/src/main/kotlin/no/ntnu/beardblaster/ui/LootDialog.kt

```diff
@@ -0,0 +1,104 @@
+package no.ntnu.beardblaster.ui
+
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.scenes.scene2d.ui.Button
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+import com.badlogic.gdx.scenes.scene2d.ui.Table
+import com.badlogic.gdx.utils.Align
+import ktx.log.debug
+import ktx.log.logger
+import ktx.scene2d.*
+import no.ntnu.beardblaster.assets.Nls
+import no.ntnu.beardblaster.commons.game.Loot
+import no.ntnu.beardblaster.game.GameInstance
+import no.ntnu.beardblaster.game.GameLoot
+import no.ntnu.beardblaster.user.UserData
+import java.util.*
+
+private val LOG = logger<LootDialog>()
+
+@Scene2dDsl
+class LootDialog(
+    val gameInstance: GameInstance
+) : Table(Scene2DSkin.defaultSkin), KTable, Observer {
+
+    var closeBtn: Button = scene2d.textButton(Nls.quitGame(), ButtonStyle.Primary.name)
+
+    companion object {
+        const val PADDING = 50f
+    }
+
+    init {
+        background = skin[Image.ModalDark]
+        pad(PADDING)
+        defaults().space(10f).align(Align.topRight)
+        columnDefaults(0).align(Align.left).expandX()
+        columnDefaults(1).align(Align.right)
+        gameInstance.gameLoot.addObserver(this)
+        if (gameInstance.gameLoot.getLoot().isNotEmpty()) {
+            renderLoot(gameInstance.gameLoot.getLoot())
+        }
+    }
+
+    private fun renderLoot(loot: List<Loot>) {
+        val userId = UserData.instance.user!!.id
+        val isWinner = userId == gameInstance.winnerWizard?.id
+        val text = if (isWinner) Nls.youWin() else Nls.youLose()
```

Comment:
Observerer at enkelte spill ender med at begge wizards får **You lose!** i loot dialogen, selv om loot distribueres korrekt mellom taper og vinner. Kan være at vi treffer på tilfeller der `gameInstance.winnerWizard?` er `null` her enkelte ganger!?

Reply:
…, men når man forfeiter et game så kommer **You lost!** og **Yuo win!** opp korrekt. 🤔

Reply:
Ja, problemet er at vi ikke sjekker for "draw" - dersom begge wizards dør i en Action-phase, så vil den første som dør bli erklært taper. Kan vurdere å legge inn sjekk for "draw" og dele ut priser deretter :-)

Reply:
Kan bekrefte at dette skjer når man spiller et spill helt til slutten. Begge får You lose, men vinneren får riktig bounty.

Reply:
Har pushet en oppdatering som håndterer *draw*.